frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -1007,6 +1008,15 @@ function BookInfo:expandString(str, file, timestamp)
                         patterns["%C"] = title
                     end
                 end
+                if patterns["%l"] then
+                    local pages_left_in_chapter = self.ui.toc:getChapterPagesLeft(pageno) or pages_left_in_chapter
```

Comment:
Whoops, I should've included this one in the other comment but that doesn't make much sense. ;-) Maybe you meant to write something else in there like pages left in book?
```suggestion
                    local pages_left_in_chapter = self.ui.toc:getChapterPagesLeft(pageno)
```